modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala

```diff
@@ -129,7 +126,7 @@ trait VoTableParser {
         .andThen(parseDoubleDegrees(VoTableParser.UCD_RA, _))
         .map(a => RightAscension.fromDoubleDegrees(a.toDoubleDegrees))
 
-    val parsePV = adapter.parseProperMotion(entries)
+    def parsePV = adapter.parseProperMotion(entries)
```

Comment:
Why did this change from `val` to `def` ?

Reply:
we don't want to eagerly load a value which may not be needed if other parts fail to validate

Reply:
`lazy val` ?